etc/module.xml

```diff
@@ -5,5 +5,5 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../lib/internal/Magento/Framework/Module/etc/module.xsd">

```

Comment:
can you please change this to `xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd"` ?